src/ShortcutPresenter.kt

```diff
@@ -151,10 +151,13 @@ public class ShortcutPresenter() : Disposable {
         if (shortcutText.isEmpty() || shortcutText == shownShortcut) return fragments
 
         when {
-            keymap.getKind() == KeymapKind.WIN -> fragments.addText(shortcutText)
-            macKeyStokesFont != null && macKeyStokesFont!!.canDisplayUpTo(shortcutText) == -1 -> {
+            keymap.getKind() == KeymapKind.WIN -> {
```

Comment:
We just don't need to access 'macKeyStrokesFont' on Mac because the default font surely can show all Mac-specific symbols. I've implemented this.